src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1695,6 +1557,157 @@ let genQuoteExpr (node: ExprQuoteNode) =
     +> genSingleTextNode node.CloseToken
     |> genNode node
 
+let genFieldsExpr (node: ExprRecordNode) =
+    col sepNln node.Fields genRecordFieldName
+
+let private genSmallRecordExpr genExtra (node: ExprRecordNode) =
```

Comment:
Because we are using a signature file, every function that is not listed inside the signature file is already private. There is no need to specify it again.